src/main/java/pwe/planner/logic/commands/PlannerRemoveCommand.java

```diff
@@ -0,0 +1,108 @@
+package pwe.planner.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static pwe.planner.logic.parser.CliSyntax.PREFIX_CODE;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javafx.collections.ObservableList;
+import pwe.planner.logic.CommandHistory;
+import pwe.planner.logic.commands.exceptions.CommandException;
+import pwe.planner.model.Model;
+import pwe.planner.model.module.Code;
+import pwe.planner.model.module.Module;
+import pwe.planner.model.planner.DegreePlanner;
+
+/**
+ * Removes module(s) from the degree plan.
+ * Related co-requisite(s) are removed as well.
+ */
+public class PlannerRemoveCommand extends Command {
+
+    public static final String COMMAND_WORD = "planner_remove";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Removes module(s) from the degree plan.\n"
+            + "Parameters: "
```

Comment:
Could you change this?
```suggestion
            + "Format: " + COMMAND_WORD + " "
```